src/config.js

```diff
@@ -22,8 +22,16 @@ const config = convict({
       format: String,
       default: '',
       env: 'FIXER_TOKEN',
-    },
+    }
   },
+  currencyLayer: {
+    token: {
+      doc: 'Access token',
+      format: String,
+      default: 'b7e9097ad7389732c12a86ecc47baeb0',
```

Comment:
better to not hardcode token,
`default: ''`
it can be added to your environment variables`